commons/com.b2international.index/src/com/b2international/index/revision/BaseRevisionBranching.java

```diff
@@ -21,12 +21,17 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 import java.util.concurrent.Callable;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.locks.ReentrantLock;
 import java.util.function.Consumer;
 
-import com.b2international.commons.exceptions.*;
```

Comment:
Revert this change please and configure your IDE to use wildcard imports on 5 or more import statements from the same package.

Reply:
Done